plugins/org.eclipse.n4js/src/org/eclipse/n4js/validation/validators/packagejson/AbstractJSONValidatorExtension.java

```diff
@@ -417,4 +431,33 @@ private boolean isValidCheckKeyMethod(Method method) {
 		}
 		return documentValues;
 	}
+
+	@Override
+	protected IssueSeverities getIssueSeverities(Map<Object, Object> context, EObject eObject) {
+		JSONDocument jsonDocument = EcoreUtil2.getContainerOfType(eObject, JSONDocument.class);
+		if (isPckjsonOfPlainJS(jsonDocument)) {
+			// switch all issues to warnings since plainjs projects do not comply to semver
```

Comment:
May be better: Retrieve the issue severity via the `getIssueSeverities` of the superclass, if the severity >= Warnings => Return warning. So that severity below warning remains unchanged?